backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -173,24 +173,23 @@ pub struct InnerLoop {
     /// How big can the queue of messages coming in to the aggregator get before messages
     /// are prioritised and dropped to try and get back on track.
     max_queue_len: usize,
+
+    /// Flag to expose the node's ip address to the feed subscribers.
+    expose_node_ips: bool,
```

Comment:
can this expose several IP addresses or just one?

i.e, the documentation and the field `expose_node_ips` is a bit contradicting ^^

Reply:
As in; it exposes the IP addresses of all connected nodes :)

Reply:
right, I still think the comment is a bit hard to understand but maybe in this context it makes sense.

Reply:
Indeed, now the comment seems a bit confusing, I guess it is coming from the plural of `ip`.

Have changed the documentation to better reflect we are referring to _all connected nodes_ as "Flag to expose the IP addresses of all connected nodes to the feed subscribers." 😄

Let me know what you think of it

Reply:
looks great now

Reply:
Thanks for the review guys! And for making this cleaner!